doc/specs/#607 - Commandline Arguments for the Windows Terminal.md

```diff
@@ -0,0 +1,724 @@
+  subsequent `new-tab` commands. When combined with any of `--maximized` or
+  `--fullscreen`, TODO: what do?
+* `[terminal_parameters]`: See [[terminal_parameters]](#terminal_parameters).
+
+
+#### `split-pane`
+
+`split-pane [--target,-t target-pane] [-h]|[-v] [--percent,-% split-percentage] [terminal_parameters]`
+
+Creates a new pane in the currently focused tab by splitting the given pane
+vertically or horizontally.
+
+**Parameters**:
+* `--target,-t target-pane`: Creates a new split in the given `target-pane`.
+  Each pane has a unique index (per-tab) which can be used to identify them.
+  These indicies are assigned in the order the panes were created. If omitted,
+  defaults to the index of the currently focused pane.
+* `-h`, `-v`: Used to indicate which direction to split the pane. `-v` is
+  "vertically" (think `[|]`), and `-h` is "horizontally" (think `[-]`). If
+  omitted, defaults to vertical. If both `-h` and `-v` are provided, defaults to
+  vertical.
+* `--percent,-% split-percentage`: Designtates the amount of space that the new
+  pane should take, as a percentage of the parent's space. If omitted, the pane
+  will take 50% by default.
+* `[terminal_parameters]`: See [[terminal_parameters]](#terminal_parameters).
+
+#### `focus-tab`
+
+`focus-tab [--target,-t tab-index]`
+
+Moves focus to a given tab.
+
+**Parameters**:
+
+* `--target,-t tab-index`: moves focus to the tab at index `tab-index`. If omitted,
+  defaults to `0` (the first tab).
+
+#### `focus-pane`
+
+`focus-pane [--target,-t target-pane]`
+
+Moves focus within the currently focused tab to a given pane.
+
+**Parameters**:
+
+* `--target,-t target-pane`: moves focus to the given `target-pane`. Each pane
+  has a unique index (per-tab) which can be used to identify them. These
+  indicies are assigned in the order the panes were created. If omitted,
+  defaults to the index of the currently focused pane (which is effectively a
+  no-op)
+
+#### `move-focus`
+
+`move-focus [--direction,-d direction]`
+
+Moves focus within the currently focused tab in the given direction.
+
+**Parameters**:
+
+* `--direction,-d direction`: moves focus in the given `direction`. `direction`
+  should be one of [`left`, `right`, `up`, `down`]. If omitted, does not move
+  the focus at all (resulting in a no-op).
+
+#### `[terminal_parameters]`
+
+Some of the preceeding commands are used to create a new terminal instance.
+These commands are listed above as accepting `[terminal_parameters]` as a
+parameter. For these commands, `[terminal_parameters]` can be any of the
+following:
+
+`[--profile,-p profile-name]|[--guid,-g profile-guid] [--startingDirectory,-d starting-directory] [commandline]`
+
+* `--profile,-p profile-name`: Use the given profile to open the new tab/pane,
+  where `profile-name` is the `name` of a profile. If `name` does not match
+  _any_ profiles, uses the default. If both `--profile` and `--guid` are
+  omitted, uses the default profile.

```

Comment:
Should `-p` and `-g` be collapsed into just `-p`? Because @mkitzan added the "read it as a guid if it is a guid, as a name if it is a name" support code and you tied it all off

Reply:
`-p {abc}` `-p Alpine`